central/globaldb/postgres.go

```diff
@@ -157,13 +172,26 @@ func CollectPostgresStats(ctx context.Context, db *pgxpool.Pool) []*stats.TableS
 		statsSlice = append(statsSlice, tableStat)
 	}
 
-	return statsSlice
+	dbStats.Tables = statsSlice
+	return dbStats
+}
+
+// CollectPostgresDatabaseStats -- collect database level stats for Postgres
+func CollectPostgresDatabaseStats(postgresConfig *pgxpool.Config) {
+	cloneSize, err := pgadmin.GetDatabaseSize(postgresConfig, migrations.CurrentDatabase)
```

Comment:
This could be misleading. If we only count the current database, the customer would see more spaces used. We can add another stat for the currently used and keep UsedBytes for the totally used.